src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements GetFigureArea {
+    private double radius = 5;
+
+    @Override
+    public String toString() {
+        return "Figure - circle: radius " + this.radius
+                + " ,area " + this.getArea() + " color "
+                + this.color;

```

Comment:
The `color` field is accessed directly here. Consider using a getter method if encapsulation is a concern, especially if the `color` field is private in the `Figure` class.